lib/charms/traefik_k8s/v1/ingress.py

```diff
@@ -69,7 +69,7 @@ def _on_ingress_revoked(self, event: IngressPerAppRevokedEvent):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 17
+LIBPATCH = 18
```

Comment:
I would add this feature to v2 only, not v1. V1 is in maintenance mode: no backports unless it's required to fix some outstanding bugs.